src/D2L.CodeStyle.TestAnalyzers/NUnit/TestAttributeAnalyzer.cs

```diff
@@ -39,7 +38,7 @@ CompilationStartAnalysisContext context
                     context: ctx,
                     types: types,
                     syntax: ctx.Node as MethodDeclarationSyntax,
-                    blacklist: whitelistedClasses
+					disallowedList: allowedClasses

```

Comment:
whitespace

Reply:
I'm not sure how this ended up changed. I'll fix it though.

Reply:
Oh, looks like it's a tabs vs. spaces thing. The rest of this file is using spaces.